android/src/main/java/com/mapbox/mapboxgl/GlobalMethodHandler.java

```diff
@@ -60,10 +61,14 @@ public void onMethodCall(MethodCall methodCall, MethodChannel.Result result) {
                 installOfflineMapTiles(tilesDb);
                 result.success(null);
                 break;
+            case "setOffline":
+                boolean offline = methodCall.argument("offline");
+                ConnectivityReceiver.instance(context).setConnected(offline ? false : null);
```

Comment:
shouldn't this be `offline ? false : true` or simply `!offline`

Reply:
`ConnectivityReceiver.setConnected`  is an override for the device state ([src](https://github.com/mapbox/mapbox-gl-native-android/blob/main/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/net/ConnectivityReceiver.java#L107))

Calling `setConnected(true)` will cause `ConnectivityReceiver` to think its permanently online.

The intention of this method is to allow forcing offline mode. The alternative would be to expose `setConnected(bool?)`, but I think that's a little confusing unless you read the docs and I can't think of many use-cases for forcing online outside of testing.

Reply:
Ok that makes sense - LGTM